docs/dataservice-write-example.md

```diff
@@ -34,6 +34,39 @@ To execute the example, run:
 
 If everything is fine, you would see `Publish Successful - TraceID: <TraceId generated by the platform>`.
 
+### Building on Android
+
+Prerequsities: make sure that you have correctly setup the Android environment (downloaded both Android SDK and NDK from official site). Also, make sure that your provided correct keyId, keySecret, and other information stated in the beginning of this README file.
+
+This example shows how to integrate and use the OLP C++ EDGE SDK in Android project.
+
+First, configure the project with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`, path to NDK's Android toolchain file set via `CMAKE_TOOLCHAIN_FILE`, and, optionally, the `ANDROID_` CMake variables, for example to configure SDK version - `ANDROID_PLATFORM`, or ABI for which project will be configured - `ANDROID_ABI`:
```

Comment:
```suggestion
First you need configure the SDK with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`, the path to NDK's Android toolchain file set via the `CMAKE_TOOLCHAIN_FILE` variable, and, optionally, the `ANDROID_` CMake variables, e.g. `ANDROID_PLATFORM` or `ANDROID_ABI`.
```

Reply:
I changed a bit this paragraph considering your changes. Please, take a look again.